README.adoc

```diff
@@ -233,38 +236,41 @@ properties is correctly handled as an error.
 To force these test cases to run in a particular order, put them in a [hotspot=testSuite file=0]`testSuite()` method and label
 it with a [hotspot=test file=0]`@Test` annotation so that it automatically runs when your test class run.
 
-Finally, the [hotspot file=1]`src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java` file
+Finally, the [hotspot file=1]`src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java` file
 is included for you to test the basic functionality of the `system` service.
 If a test failure occurs, then you might have introduced a bug into the code.
 
-InventoryEndpointTest.java
+InventoryEndpointIT.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointIT.java[]
 ----
 
-SystemEndpointTest.java
+SystemEndpointIT.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointIT.java[]
 ----
 
 // =================================================================================================
 // Running the tests
 // =================================================================================================
 
-[role='command']
-include::{common-includes}/mvnverify.adoc[]
+=== Running the tests
+
+Since you started Open Liberty in development mode at the start of the guide, press the `enter/return` key to run the tests.
+
+If the tests pass, you will see a similar output to the following:
```

Comment:
Change
"If the tests pass, you will see a similar output to the following:"
To
"If the tests pass, you see a similar output to the following example:"